docs/src/content/docs/tr/reference/overrides.md

```diff
@@ -0,0 +1,375 @@
+#### `entryMeta`
+
+**Tür:** `{ dir: 'ltr' | 'rtl'; lang: string }`
+
+Sayfa içeriği için bölge metaverisi. Bir sayfa, fallback içerik kullanıyorsa, sayfa içeriğinin baş düzey bölge değerlerinden farklı olabilir.
+
+#### `entry`
+
+Mevcut sayfa için Astro içerik koleksiyonu girişi. 
+`entry.data` altındaki mevcut sayfa için frontmatter değerlerini içerir.
+
+```ts
+entry: {
+  data: {
+    title: string;
+    description: string | undefined;
+    // vb.
+  }
+}
+```
+
+Bu nesnenin şekli hakkında daha fazla bilgiyi [Astro'nun Koleksiyon Giriş Türü](https://docs.astro.build/en/reference/api-reference/#collection-entry-type) başvurusunda bulabilirsiniz.
+
+#### `sidebar`
+
+**Tür:** `SidebarEntry[]`
+
+Bu sayfa için site gezinme kenar çubuğu girişleri.
+
+#### `hasSidebar`
+
+**Tür:** `boolean`
+
+Bu sayfada kenar çubuğunun gösterilip gösterilmeyeceğini belirler.
+
+#### `pagination`
+
+**Tür:** `{ prev?: Link; next?: Link }`
+
+Kenar çubuğundaki önceki ve sonraki sayfa bağlantıları.
+
+#### `toc`
+
+**Tür:** `{ minHeadingLevel: number; maxHeadingLevel: number; items: TocItem[] } | undefined`
+
+Bu sayfa için içerik tablosu, açıkken.
+
+#### `headings`
+
+**Tür:** `{ depth: number; slug: string; text: string }[]`
+
+Mevcut sayfadaki tüm Markdown başlıklarının dizisi. 
+Starlight'ın yapılandırma seçeneklerini dikkate alan bir içerik tablosu bileşeni oluşturmak istiyorsanız, [`toc`](#toc) yerine kullanın.
+
+#### `lastUpdated`
+
+**Tür:** `Date | undefined`
+
+Etkinleştirilmişse, bu sayfanın son ne zaman güncellendiğini temsil eden JavaScript `Date` nesnesi.
+
+#### `editUrl`
+
+**Tür:** `URL | undefined`
+
+Eğer etkinse, bu sayfanın düzenlenebileceği adres için bir `URL` nesnesi.
+
+---
+
+## Components
+
+### Head
+
+These components are rendered inside each page’s `<head>` element.
+They should only include [elements permitted inside `<head>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head#see_also).
+Bu bileşenler her sayfanın `<head>` etiketi içinde oluşturulur. 
+Yalnızca [`<head>` içine izin verilen öğeleri](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head#see_also) içermelidir.
+
+#### `Head`
+
+**Varsayılan component:** [`Head.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/Head.astro)
+
+Her sayfanın `<head>` etiketi içinde oluşturulan bileşen. 
+`<title>` ve `<meta charset="utf-8">` gibi önemli etiketleri içerir.
+
+Bu bileşeni son çare olarak geçersiz kılın. 
+Mümkünse Starlight yapılandırma seçeneği olan [`head`](/tr/reference/configuration#head) seçeneğini tercih edin.
+
+#### `ThemeProvider`
+
+**Varsayılan component:** [`ThemeProvider.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/ThemeProvider.astro)
+
+Karanlık/aydınlık tema desteği oluşturan bileşen `<head>` içinde oluşturulan bileşen. 
+Varsayılan uygulama, inline betik ve [`<ThemeSelect />`](#themeselect) için kullanılan bir `<template>` içerir.
+
+---
+
+### Accessibility
```

Comment:
Just to double check, is this a choice to not translate this title?

Reply:
Erişebilirlik